Source/i_video.c

```diff
@@ -939,7 +939,12 @@ void I_FinishUpdate(void)
    // [AM] Figure out how far into the current tic we're in as a fixed_t.
    if (uncapped)
    {
-	fractionaltic = I_GetTimeMS() * TICRATE % 1000 * FRACUNIT / 1000;
+        const double tics_per_msec = TICRATE / 1000.0f;
+
+        int tic_time = I_TickElapsedTime();
+
```

Comment:
Do you think we could get away without the float by just writing out tics_per_msec into the fractionaltic formula?

Reply:
Do you mean this: `fractionaltic = (fixed_t) (tic_time * FRACUNIT * (TICRATE / 1000.0f));`? It will be float anyway.

Reply:
I guess we can use fixed point:
```
        const fixed_t tics_per_msec = FixedDiv(TICRATE << FRACBITS, 1000 << FRACBITS);

        int tic_time = I_TickElapsedTime();

        fractionaltic = FixedMul(tic_time << FRACBITS, tics_per_msec);
        fractionaltic = BETWEEN(0, FRACUNIT, fractionaltic);
```

Reply:
`fractionaltic = tic_time * FRACUNIT * TICRATE / 1000`?

Reply:
Hmm, I think I messed up somewhere before, now it works.